drivers/gpu/drm/tests/drm_dp_mst_helper_test.c

```diff
@@ -5,15 +5,15 @@
 
 #define PREFIX_STR "[drm_dp_mst_helper]"
 
+#include <kunit/test.h>
 #include <linux/random.h>
 
 #include <drm/display/drm_dp_mst_helper.h>
 #include <drm/drm_print.h>
 
 #include "../display/drm_dp_mst_topology_internal.h"
-#include "test-drm_modeset_common.h"
```

Comment:
🤔

Reply:
This file is used to hold the signature of the tests. With KUnit, it becomes useless.